pkg/gooffensive/token_windows.go

```diff
@@ -0,0 +1,70 @@
+package gooffensive
+
+import (
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+const (
+	SE_PRIVILEGE_ENABLED = uint32(0x00000002)
+)
+
+// SePrivEnable adjusts the privileges of the current process to add the passed in string. Good for setting 'SeDebugPrivilege'
+func SePrivEnable(s string) error {
+	type LUID struct {
+		LowPart  uint32
+		HighPart int32
+	}
+	type LuidAndAttributes struct {
+		Luid       LUID
+		Attributes uint32
+	}
+	type TokenPrivileges struct {
+		PrivilegeCount uint32
+		Privileges     [1]LuidAndAttributes
+	}
+
+	modadvapi32 := windows.NewLazySystemDLL("advapi32.dll")
```

Comment:
I think we should export all the DLL loading to a separate package/file (`syscallz_windows.go` somewhere?), since we're going to repeat these steps (load DLL + resolve function name) a lot.
Thus we'll be able to do something like `syscallz.AdjustTokenPrivileges`. Thoughts on this?

Reply:
In the spirit of making the code copy-paste friendly (since I certainly will be wanting to hard-code a lot of the functions), I think I'd like to avoid that for now. AV will eventually just pick up the loaded package string, making anything that imports this lib very easy to detect, so exported functions should be reasonably standalone (and resolving function names isn't a massive overhead iirc). I'll think about it a bit more, though, since that will certainly make the code tidier